environment/src/main/java/com/sequenceiq/environment/environment/encryption/EncryptionResourcesNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package com.sequenceiq.environment.environment.encryption;
+
+import com.sequenceiq.cloudbreak.common.exception.CloudbreakServiceException;
+
+public class EncryptionResourcesNotFoundException extends CloudbreakServiceException {
```

Comment:
this is probably requires ExceptionMapper right @topolyai5 ?

Reply:
Yes, mapper required

Reply:
`CloudbreakServerExceptionMapper` exists for `CloudbreakServiceException`